webapp/src/index.tsx

```diff
@@ -833,11 +781,15 @@ export default class Plugin {
         this.unsubscribers.push(store.subscribe(() => {
             const currentChannelId = getCurrentChannelId(store.getState());
             if (currChannelId !== currentChannelId) {
+                const firstLoad = !currChannelId;
                 currChannelId = currentChannelId;
 
-                fetchChannelData(currChannelId).then((actions) =>
-                    store.dispatch(batchActions(actions)),
-                );
```

Comment:
I am removing this as I don't think it's needed anymore. We fetch all channels anyway on first load and then rely on websocket to keep things up to date. I couldn't see any side effect so far but will need to monitor.